### PR TITLE
Review controls.

### PR DESCRIPTION
**Pull request by mariaterzieva12**
This pull request contains the review of several controls including Rating, SegmentedControl, Popup, NumericInput, Path, ProgressBar, SideDrawer, SignaturePad, and SlideView controls.

### CODE REVIEW

**Comment by mariaterzieva12** [comment]
RichTextEditor and TabView control are included in the pull request.

File: knowledge-base/datagrid-busy-indicator-color.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Change the BusyIndicator color in the DataGrid control.
+description: This article shows how to change the BusyIndicator color and animation, when DataGrid IsBusy property is true.
```
Comment (yordan-mitev) [inline knowledge-base/datagrid-busy-indicator-color.md:3]:
```suggestion
description: This article shows how to change the BusyIndicator color and animation when the DataGrid's IsBusy property is set to true.
```

File: knowledge-base/datagrid-busy-indicator-color.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Description
+
+This article shows how to change the BusyIndicator color and animation, when DataGrid `IsBusy` property is true.
+
+
+
+## Solution 
+
+
```
Comment (yordan-mitev) [inline knowledge-base/datagrid-busy-indicator-color.md:37]: I guess you will add the content later?

File: controls/progressbar/indeterminate-mode.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ slug: progressbar-indeterminate-mode
 
 Telerik ProgressBar for MAUI supports two modes of operation:
```
[the remark this answers is not in the document]
Comment (mariaterzieva12) [reply controls/progressbar/indeterminate-mode.md:11]: These are the modes of the ProgressBar control, they are not API and shouldn't be added as code. With bold we want to pay attention to these two modes.

[the remark this answers is not in the document]
Comment (yordan-mitev) [reply controls/progressbar/indeterminate-mode.md:11]: You don't have to use monospace for them, but bold is also not acceptable. They must be in the default font.

**Review by didiyordanova — APPROVED** [review]

**Pull request merged**